<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QMainWindow" name="SimulationWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>911</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ENKA-S</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/MainWindow/img/icon.png</normaloff>:/MainWindow/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Sunken</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ParticleRenderer" name="oglParticleRenderer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="btnChangeSpeed">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/camera.png</normaloff>../img/camera.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblTime">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Wird geladen...</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnJumpToStart">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/left_double.png</normaloff>../img/left_double.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnStepBack">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/left.png</normaloff>../img/left.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnPlayStop">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/play.png</normaloff>../img/play.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnStepForward">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/right.png</normaloff>../img/right.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnJumpToEnd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../img/right_double.png</normaloff>../img/right_double.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblFPS">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="wgtSidebar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="elideMode">
            <enum>Qt::TextElideMode::ElideNone</enum>
           </property>
           <widget class="DiagnosticsTab" name="tabDiagnostics">
            <attribute name="title">
             <string>Diagnostik</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Analyse</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="RenderSettingsWidget" name="wgtSettings" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>270</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="btnToggleSidebar">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::ArrowType::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btnMovie">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/SimulationWindow/img/movie.png</normaloff>:/SimulationWindow/img/movie.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnScreenshot">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/SimulationWindow/img/camera.png</normaloff>:/SimulationWindow/img/camera.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnToggleSettings">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/SimulationWindow/img/gear.png</normaloff>:/SimulationWindow/img/gear.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::ArrowType::NoArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="hslNavigation">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    height: 8px; /* Thickness of the groove */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #CCCCCC, stop:1 #CCCCCC); /* Background gradient of the groove */
}

QSlider::sub-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #77A8D6, stop:1 #77A8D6); /* Color of the groove after the handle */
}

QSlider::add-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #DDDDDD, stop:1 #DDDDDD); /* Color of the groove before the handle */
}

QSlider::handle:horizontal {
    background: #333333; /* Color of the handle */
    width: 12px; /* Width of the handle */
    height: 20px; /* Height of the handle */
    margin-top: -6px; /* Adjust the vertical position of the handle */
    margin-bottom: -6px; /* Adjust the vertical position of the handle */
}

QSlider::handle:horizontal:hover {
    background: #666666; /* Color of the handle */
}</string>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="pageStep">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderSettingsWidget</class>
   <extends>QWidget</extends>
   <header>render_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DiagnosticsTab</class>
   <extends>QWidget</extends>
   <header>diagnostics_tab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParticleRenderer</class>
   <extends>QOpenGLWidget</extends>
   <header>particle_renderer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
